ipc/src/lib.rs

```diff
@@ -33,15 +34,20 @@ extern crate mio;
 #[cfg(not(windows))]
 extern crate bytes;
```

Comment:
Some crates are also imported directly in `ipc/src/uds.rs`, might be good to keep everything in a single place.